### PR TITLE
unit test for kubectl config unset

### PR DESCRIPTION
**Pull request by zjj2wry**
**What this PR does / why we need it**:
unit test for kubectl config unset

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
think you 

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by ericchiang** [comment]
@k8s-bot ok to test

File: pkg/kubectl/cmd/config/unset_test.go [hunk 1]
```diff
@@ -0,0 +1,128 @@
+/*
```
Comment (ericchiang) [inline pkg/kubectl/cmd/config/unset_test.go]: The formatting for this header is wrong. Please see https://github.com/zjj2wry/kubernetes/blob/05ccf5bc6522e60533dbf7eeb41589903096c8d0/pkg/kubectl/cmd/config/get_clusters_test.go#L1-L15

(though this one should still be marked 2017)

**Comment by ericchiang** [comment]
@zjj2wry thanks can you squash your commits?

**Comment by zjj2wry** [comment]
@ericchiang ok o.o

**Comment by zjj2wry** [comment]
@ericchiang

**Comment by ericchiang** [comment]
cc @kubernetes/sig-cli-pr-reviews can someone give this a lgtm?

**Comment by fabianofranz** [comment]
@zjj2wry thanks for the pr!
/release-note-note
/approve
/lgtm

**Pull request merged**